estuary/utils/recents.py

```diff
@@ -16,20 +16,32 @@ def get_recent_nodes():
     """
     Get the most recent nodes of each node type.
 
-    :return: a dictionary with the keys as names of each node type, and
-        values of arrays of the most recents of each node
-    :rtype: dict
+    :return: a tuple with the first value as a dictionary with the keys as 
+        names of each node type, and values of arrays of the most recents of
+        each node, and the second as metadata
+    :rtype: tuple
     """
-    label_dict = {
+    timestamp_dict = {
         FreshmakerEvent.__label__: 'id',
         BugzillaBug.__label__: 'modified_time',
         DistGitCommit.__label__: 'commit_date',
         KojiBuild.__label__: 'completion_time',
         Advisory.__label__: 'update_date'
     }
+    id_dict = {
```

Comment:
This should just be an empty dictionary since your code below will create these entries even if they don't already exist.